apps/astarte_data_updater_plant/lib/astarte_data_updater_plant/amqp_vhost_supervisor.ex

```diff
@@ -0,0 +1,44 @@
+defmodule Astarte.DataUpdaterPlant.VHostSupervisor do
+  use DynamicSupervisor
+  require Logger
+
+  alias Astarte.DataUpdaterPlant.AMQPTriggersProducer
+  @impl true
+  def init(_init_arg) do
+    Logger.info("AMQPTriggers dynamic supervisor init.", tag: "amqp_triggers_sup_init")
+    DynamicSupervisor.init(strategy: :one_for_one)
+  end
+
+  def start_link(init_arg) do
+    DynamicSupervisor.start_link(__MODULE__, init_arg, name: __MODULE__)
+  end
+
+  def start_for_realm(realm_name) do
```

Comment:
this reuses existing process so `start` is misleading

```suggestion
  def for_realm(realm_name) do
```